kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/sidePanels/QuizResourceSelection/index.vue

```diff
@@ -426,7 +426,7 @@
         },
         search: {
           filters: {
-            kind_in: [ContentNodeKinds.EXERCISE, ContentNodeKinds.TOPIC],
+            kind: ContentNodeKinds.EXERCISE,
```

Comment:
Is this the desired behavior? In the quiz side panel figma specs we can see and browse search results folders https://www.figma.com/design/lVJt5ukOrxS9qay0rXy7GC/0.18-Coach-updates?node-id=2189-184132&t=SCJQdMgJhYMrDRmv-0.

Just checkin-in :sweat_smile:

When fetching quizzes for topicTree this also happens but the `annotateTopicsWithDescendantCounts` filters the folders that doesnt have any exercises inside.

Reply:
Perhaps the [search](https://github.com/learningequality/kolibri/blob/47e46a18cf227d87df884e4940fe8f9a131d48b1/kolibri/plugins/coach/assets/src/composables/useResourceSelection.js#L26-L27) property in the useResourceSelection param object should also take in an annotator into which we could pass in that `annotateTopics...` function and call in the fetch function where needed?

Reply:
I had discussed this with @marcellamaki - the specs introduced a difference from the current search behaviour in 0.17 by adding the ability to see folders in the search results.

The alternative is to have to add some gnarly backend updates in order to be able to filter folders by whether they contain descendant folders.

We can get away with not doing this for the topic tree navigation because of our "annotateTopics" trick which allows us to filter out folders that have no descendant assessments, however for search, this would allow us to still produce searches that have zero results (which is another issue that was being fixed in the course of this work).

All in all, it seemed simpler to not change the previous behaviour in this respect.

Reply:
Thanks for that Richard. 

Noting I'll do manual QA today to give the  :heavy_check_mark: